iotdb/src/main/java/org/apache/iotdb/db/writelog/recover/ExclusiveLogRecoverPerformer.java

```diff
@@ -316,9 +314,8 @@ private void replayLog() throws RecoverException {
     try {
       FileNodeManager.getInstance().closeOneFileNode(writeLogNode.getFileNodeName());
     } catch (FileNodeManagerException e) {
-      logger.error("Log node {} cannot perform flush after replaying logs! Because {}",
-          writeLogNode.getIdentifier(), e.getMessage());
-      throw new RecoverException(e);
+      throw new RecoverException(String.format("Log node %s cannot perform flush after replaying logs! Because %s",
+          writeLogNode.getIdentifier(), e.getMessage()), e);
```

Comment:
I am not sure if the e.getMessage() is redundant here.

Reply:
> I am not sure if the e.getMessage() is redundant here.
This is the javadoc of Exception(String message, Throwable cause):

> Constructs a new exception with the specified detail message and cause.
> Note that the detail message associated with cause is not automatically incorporated in this exception's detail message.

So, I think Tian is correct.

Reply:
>I am not sure if the e.getMessage() is redundant here.

This is the javadoc of Exception(String message, Throwable cause):

>Constructs a new exception with the specified detail message and cause.
>Note that the detail message associated with cause is not automatically incorporated in this exception's detail message.

So, leaving the concat of the detailed message with cause.getMessage() to the exception handler is more neat, I think.

Reply:
If the programmer only wants to know the cause of the exception, this may help him find out without looking down to the lower exceptions.
While this is not necessary, I will remove them for consistency.